docs/rst/manual/reference/configuration/keywords.rst

```diff
@@ -681,19 +680,33 @@ and/or history matching project.
 
         **Parameter fields**
 
-        A parameter field (e.g. porosity or permeability) is defined as follows:
+        A parameter field (e.g. porosity or permeability or Gaussian Random Fields from APS) is defined as follows:
 
         ::
 
-                FIELD  ID PARAMETER   <ECLIPSE_FILE>  INIT_FILES:/path/%d  MIN:X MAX:Y OUTPUT_TRANSFORM:FUNC INIT_TRANSFORM:FUNC
+                FIELD  ID PARAMETER   <ECLIPSE_FILE>  INIT_FILES:/path/%d  MIN:X MAX:Y OUTPUT_TRANSFORM:FUNC INIT_TRANSFORM:FUNC  FORWARD_INIT:True
 
-        Here ID is again an arbitrary string, ECLIPSE_FILE is the name of the file ERT
-        will export this field to when running simulations. Note that there
-        should be an IMPORT statement in the ECLIPSE data file corresponding to the
-        name given with ECLIPSE_FILE. INIT_FILES is a filename (with an embedded %d)
+	Here ID must be the same as the name of the parameter in the INIT_FILES.
+        ECLIPSE_FILE is the name of the file ERT will export this field to when 
+        running simulations. Note that there should be an IMPORT statement in 
+        the ECLIPSE data file corresponding to the name given with ECLIPSE_FILE in case
+        the field parameter is a field used in ECLIPSE data file like perm or poro. 
+        INIT_FILES is a filename (with an embedded %d if FORWARD_INIT is set to False)
         to load the initial field from. Can be RMS ROFF format, ECLIPSE restart format
         or ECLIPSE GRDECL format.
 
+        FORWARD_INIT:True means that the files specified in the INIT_FILES are expected 
+        to be created by some FORWARD model, and does not need any embedded %d.
```

Comment:
```suggestion
        to be created by the forward model, and does not need any embedded %d.
```

Reply:
I got some comments that perhaps I should take out APS related documentation. Now APS and also necessary documentation how to get it work together with ERT  (related to the GRID and FIELD keywords) is described in  the APS documentation, but still I get questions from users how should they set it up to work with ERT. That is why I included an example for how to use FIELD keyword with APS and the comments about the ERTBOX grid that is used by FMU and ERT and RMS APSGUI.
What do you think: Am I mixing too much APS application into the ERT documentation for the FIELD keyword and what should I remove from the ERT documentation?

Reply:
My 2 cents on this:

First, ideally I would have preferred to see no external tooling appear in the documentation and instead in examples or tutorials. However, certain tools (like Eclipse) is currently so interconnected with the tool that even though it is not part of ERT, nor openly available it is still mentioned frequently in the documentation. Hiding this connection in the documentation makes it no less true in reality and only makes the current documentation less usable.

Second, double documentation of something makes for the double amount of documentation to maintain. If this is already described in the APS documentation then I think that is a good reason for not including it here as well.

Third, if the real issue is discoverability of the APS documentation internally I suggest we start by adding a pointer to it in FMU-docs (https://fmu-docs.equinor.com).

Reply:
I reduce the pure APS related documentation under the FIELD command. I will check FMU doc for references to APS documentation.

Reply:
I have added some comments to the use of the GRID keyword related to the FIELD keyword. It explains the new way of handling FIELD parameters by using a help grid called ERTBOX grid which is used to define the dimensions of the fields specified in the FIELD keyword. The new way of handling this is to define one field parameter per type of parameter per modelling zone instead of one parameter per type for the whole modelling grid. It avoids resampling problems and is now implemented as the standard way of exchanging field data between APS and ERT in FMU projects.